load_balancers.go

```diff
@@ -39,6 +42,7 @@ type LoadBalancer struct {
 	Region              *Region          `json:"region,omitempty"`
 	DropletIDs          []int            `json:"droplet_ids,omitempty"`
 	Tag                 string           `json:"tag,omitempty"`
+	Tags                []*Tag           `json:"tags,omitempty"`
```

Comment:
I think we want this to be just `[]string` like [in](https://github.com/digitalocean/godo/blob/master/kubernetes.go#L96) [other](https://github.com/digitalocean/godo/blob/master/droplets.go#L61) [places](https://github.com/digitalocean/godo/blob/master/images.go#L48).

Reply:
👇